src/ui/AriaLive/AriaLive.ts

```diff
@@ -18,7 +18,8 @@ export class AriaLive implements IAriaLive {
   }
 
   public updateText(text: string) {
-    $$(this.ariaLiveEl).text(text);
+    const liveText = text === $$(this.ariaLiveEl).text() ? `${text} \u00A0` : text;
```

Comment:
allows to mention the same text twice in a row e.g. 
- Searching for "q"
- Says "No values found"
- Searching for "qa"
- Says "No values found" another time